### PR TITLE
Surface Web Host configuration in WS topic

### LINKED ISSUE #7842: Changing the port number
The functionality worked OK but I cannot determine how to run as a windows service and use a none default port number

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b0bd8cb8-7e28-8c4d-a550-3aea8f3ab4dd
* Version Independent ID: 4b60d24e-ef05-15e5-10b4-483094081bf3
* Content: [Host ASP.NET Core in a Windows Service](https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/windows-service?view=aspnetcore-2.1)
* Content Source: [aspnetcore/host-and-deploy/windows-service.md](https://github.com/aspnet/Docs/blob/master/aspnetcore/host-and-deploy/windows-service.md)
* Product: **aspnet-core**
* GitHub Login: @guardrex
* Microsoft Alias: **tdykstra**

### PR DESCRIPTION
**Pull request by guardrex**
Fixes #7842

### CODE REVIEW

File: aspnetcore/host-and-deploy/windows-service.md [hunk 1]
```diff
@@ -215,8 +215,9 @@ If the custom `WebHostService` code requires a service from dependency injection
 
 ## Proxy server and load balancer scenarios
 
-Services that interact with requests from the Internet or a corporate network and are behind a proxy or load balancer might require additional configuration. For more information, see [Configure ASP.NET Core to work with proxy servers and load balancers](xref:host-and-deploy/proxy-load-balancer).
+Services that interact with requests from the Internet or a corporate network and are behind a proxy or load balancer might require additional configuration. For more information, see <xref:host-and-deploy/proxy-load-balancer>.
```
Comment (scottaddie) [inline aspnetcore/host-and-deploy/windows-service.md]: The leading "S" is missing on this 1st sentence:
ervices --> Services

Comment (guardrex) [reply aspnetcore/host-and-deploy/windows-service.md]: @scottaddie Where? I don't see it on Line 218. Another line?

Comment (scottaddie) [reply aspnetcore/host-and-deploy/windows-service.md]: Nevermind. This must be a bug in the diff view. Here's what I see:

image

Comment (guardrex) [reply aspnetcore/host-and-deploy/windows-service.md]: Interesting ... I'm good over here. Maybe that's reportable to GH, but I guess we can just keep an :eye: out for it to see if it happens a few times.

capture

**Pull request merged**